ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -79,6 +79,55 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: aws-ipi-disconnected-private-p1-f4
```

Comment:
I think `f4` is a bit aggressive, can should lower its frequency?

Reply:
there is only one disconnected profile triggerd by  f4(weekly), I am ok with it, or could you give a suggest frequency based on your experience?

Reply:
Yeah, f14 is okay for me. BTW, another issue I proposed to remove `aws-ipi-p2-f14` one, because is being well covered by others.